src/olympia/blocklist/admin.py

```diff
@@ -155,31 +158,76 @@ def review_unlisted(self, obj):
                 '<a href="{}">{}</a>', url, _('Review Unlisted'))
         return ''
 
+    def url_link(self, obj):
+        return format_html('<a href="{}">{}</a>', obj.url, obj.url)
+
+    def block_history(self, obj):
+        history_format_string = (
+            '<li>{}. {} {} Versions {} - {}<ul><li>{}</li></ul></li>')
+
+        logs = ActivityLog.objects.for_addons((obj.addon,)).order_by('created')
+        log_entries_gen = (
+            (log.created.date(),
+             log.user.name,
+             str(log),
+             log.details['min_version'],
+             log.details['max_version'],
+             log.details['reason'])
+            for log in logs)
```

Comment:
Am I missing something ? You are not filtering on `action`, so you're going to get a bunch of random `ActivityLog` objects that are irrelevant to this, and would not have the keys you're looking for in `log.details`.

Reply:
ugh, you're right.  It worked locally!  (when I was blocking the addons created during initialization that wouldn't have any other ActivityLog records...)

Reply:
Does fail locally with `KeyError` because of this.

Reply:
https://github.com/mozilla/addons-server/pull/12816/commits/be1dec6bfa4cf4cc8f951e3bd7d20c77beff8f78